### PR TITLE
Jabark/iot central environments

### PR DESCRIPTION
**Pull request by Jackbk**
Updated iot central commands to allow the api for token collection to be overridden.  This will allow us to use this extension in dev and ppe environments.

---
This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

Thank you for contributing to the IoT extension!

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] If introducing new functionality or modified behavior, are they backed by unit and integration tests?
- [ ] In the same context as above are command names and their parameter definitions accurate? Do help docs have sufficient content?
- [ ] Have **all** unit **and** integration tests passed locally? i.e. `pytest -s <project root>/azext_iot/tests`
- [ ] Have static checks passed using the .pylintrc and .flake8 rules? Look at the CI scripts for example usage.

### CODE REVIEW

File: azext_iot/_params.py [hunk 1]
```diff
@@ -412,6 +412,9 @@ def load_arguments(self, _):
         context.argument('repair', options_list=['--repair', '-r'],
                          arg_type=get_three_state_flag(),
                          help='Reinstall uamqp dependency compatible with extension version. Default: false')
+        context.argument('central_api_uri', options_list=['--central-api-uri'],
+                         help='IoT Central API override.  For use with environments other than production. '
+                         'Default: api.azureiotcentral.com.')
```
Comment (digimaun) [inline azext_iot/_params.py]: Are you sure you need to add the 'Default: api.azureiotcentral.com' part of the string? The knack arg parser should automatically add the default value if its assigned to an arg in the command bound function. Please double check.

Comment (digimaun) [reply azext_iot/_params.py]: The other explicit 'Default: blah' are for some limited cases *usually* for the 'three state flag' args i.e. when you require switches/flags.

File: azext_iot/_params.py [hunk 2]
```diff
@@ -420,9 +423,15 @@ def load_arguments(self, _):
     with self.argument_context('iotcentral device-twin show') as context:
```
Comment (digimaun) [inline azext_iot/_params.py]: Just curious is there a removal of the deprecated 'iotcentral' in favor of 'iot central' in the works?

File: azext_iot/_params.py (hunk 1 above)
Comment (Jackbk) [reply azext_iot/_params.py]: Yep you are right, I have removed and tested it.  Thanks!

File: azext_iot/_params.py (hunk 2 above)
Comment (Jackbk) [reply azext_iot/_params.py]: I asked about this, it's still planned but no work has started on it yet.  The IoT Central team is focusing on stabilizing the ignite features first

**Pull request merged**